rdrf/rdrf/patients_listing.py

```diff
@@ -264,7 +264,17 @@ def apply_custom_ordering(self, qs):
         if key_func:
             # we have to retrieve all rows - otherwise , queryset has already been
             # ordered on base model
-            return sorted(qs, key=key_func[0], reverse=(self.sort_direction == "desc"))
+            logger.debug("key = %s" % key_func[0])
+
+            def key_func_wrapper(thing):
+                value = key_func[0](thing)
+                if value is None:
+                    return ""
```

Comment:
That is correct so if None was in a column for a date it would fail - we're mandating date of birth though. 
I could make a type map of the columns I suppose